scripts/events.py

```diff
@@ -99,6 +98,43 @@ def one_moon(self):
         # age up the clan, set current season
         game.clan.age += 1
         get_current_season()
+
+        # update afterlife tempers
+        for c in game.updated_afterlife_cats:
+            if not c.status.newly_joined_group:
+                continue
+            if (
+                not c.status.rank
+                in (
```

Comment:
```suggestion
                c.status.rank
                not in (
```
Same thing but a bit easier to read.